cypress/support/fragments/requests/requestDetail.js

```diff
@@ -152,6 +173,30 @@ export default {
     cy.expect(cancelRequestButton.exists());
   },
 
+  verifyAllRequestOptionsDisplayed() {
+    cy.expect([
+      cancelRequestButton.exists(),
+      moveRequestButton.exists(),
+      duplicateRequestButton.exists(),
+      editRequestButton.exists(),
+      reorderQueueButton.exists(),
+    ]);
+  },
```

Comment:
There is a method created for this verification already, check `verifyActionsAvailableOptions` method in requestDetails.js